templates/company/filing_history/view_content_certified.html.tx

```diff
@@ -62,7 +62,7 @@
                       % }
                       % if ($item._missing_message == 'available_in_5_days') {
                       <div class="form-hint">
-                        This document is being processed and will be available in 5 days.
+                        This document is being processed and will be available in <% $recently_filed %> days.
```

Comment:
For the certified template you'll have to do the same as you've done in `FilingHistory.pm` and stash the `recently_filed` variable in this file https://github.com/companieshouse/ch.gov.uk/blob/master/lib/ChGovUk/Controllers/Company/CertifiedDocuments.pm#L44